src/Microsoft.CodeAnalysis.Razor.Workspaces/RazorDocumentExcerptService.cs

```diff
@@ -89,24 +100,26 @@ public RazorDocumentExcerptService(DocumentSnapshot document, ISpanMappingServic
                 classifiedSpans[i] = new ClassifiedSpan(classifiedSpan.ClassificationType, updated);
             }
 
-            return new ExcerptResult(excerptText, excerptSpan, classifiedSpans.ToImmutable(), document, span);
+            return new ExcerptResultInternal(excerptText, primarySpan, classifiedSpans.ToImmutable(), document, span);
         }
-
-        private TextSpan ChooseExcerptSpan(SourceText primaryText, TextSpan primarySpan, ExcerptMode mode)
+        
+        private TextSpan ChooseExcerptSpan(SourceText primaryText, TextSpan primarySpan, ExcerptModeInternal mode)
         {
             var startLine = primaryText.Lines.GetLineFromPosition(primarySpan.Start);
             var endLine = primaryText.Lines.GetLineFromPosition(primarySpan.End);
 
-            // If we're showing a single line then this will do. Otherwise expand the range by 1 in
+            // If we're showing a single line then this will do. Otherwise expand the range by 3 in
```

Comment:
Why the change from 1 to 3? Is that what Roslyn does in other places?

Reply:
Yeah, HeeJae recommend 3.